rails/db/migrate/20190601202823_migrate_place_id_story_idto_join_table_place_story.rb

```diff
@@ -0,0 +1,11 @@
+class MigratePlaceIdStoryIdtoJoinTablePlaceStory < ActiveRecord::Migration[5.2]
+  def change
+    points = Point.all.index_by(&:place_id)
+
+     Place.find_each do |place|
+      points = Point.where(place_id: place.id).map(&.id)
```

Comment:
Should this be
`Point.where(place_id: place.id).map(&:id)` ?

I was getting some syntax errors when trying to run the migrations.